index.html

```diff
@@ -19,12 +19,20 @@
     <main>
       <section class="landing">
         <header class="nav_bar">
+          <menu class="menu_items">
+            <p class="close_menu_icon">X</p>
+            <ul>
+              <li><a href="#portfolio">Portfolio</a></li>
+              <li><a href="#contact">Contact</a></li>
+              <li><a href="#about">About</a></li>
+            </ul>
+          </menu>
           <div class="toolbar">
             <a href="#home" class="welcome">Welcome</a>
             <ul>
               <li><a href="#portfolio">Portfolio</a></li>
               <li><a href="#contact">Contact</a></li>
-              <li><a href="#contact">About</a></li>
+              <li><a href="#about">About</a></li>
             </ul>
```

Comment:
- [ ] the link to the contact form is not working because you are directed to an element with id "contact" however the contact form has an id of "contact_form", please make sure to fix this before requesting another review.